app/controllers/document_exports_controller_base.rb

```diff
@@ -101,6 +101,9 @@ def download
       'HmisDataQualityTool::DocumentExports::ReportByClientExcelExport',
       'SystemPathways::DocumentExports::ReportExport',
       'SystemPathways::DocumentExports::ReportExcelExport',
+      'DocumentExports::HealthPctpPdfExport',
+      'DocumentExports::HealthPctpSignaturePdfExport',
+      'DocumentExports::HealthCaPdfExport',
```

Comment:
Should these be in their drivers?

Reply:
@awisema They are in the drivers but I didn't add the extra level in the model folder. I can add that if you think it is important.